bootstrap/helpers/parsers.php

```
@@ -16,6 +16,132 @@
+ * @throws \Exception If the volume string contains command injection attempts
+ */
+function validateVolumeStringForInjection(string $volumeString): void
+{
+    // Parse the volume string to extract source and target
+    $parts = explode(':', $volumeString);
+    if (count($parts) < 2) {
+        // Named volume without target - only validate the name
+        try {
+            validateShellSafePath($parts[0], 'volume name');
+        } catch (\Exception $e) {
+            throw new \Exception(
```

PHPMD (2.15.0)

Missing class import via use statement (line '109', column '23'). (undefined)

(MissingImport)